platform/docker-compose-nightly.yml

```diff
@@ -0,0 +1,174 @@
+version: '3.5'
+
+networks:
+  frontend:
+    name: frontend
+  storage:
+    name: storage
+
+services:
+  traefik:
+    image: traefik:v1.7
+    command: --api --docker
+    ports:
+      # only expose https to outside world
+      - "443:443"   # SSL
+      - "8080:8080" # Traefik dashboard
```

Comment:
useless without the UI enabled for Traefik

Reply:
Ok I will remove the port 8080